src/common/services/api/cart.service.js

```diff
@@ -24,21 +24,27 @@ import { startMonth } from '../giftHelpers/giftDates.service'
 
 const cartTotalCookie = 'giveCartItemCount'
 const cartTotalCookieDomain = 'cru.org'
+const brandedCheckoutDomains = ['secure.cru.org']
```

Comment:
I wonder if non-branded checkout domains would be more maintainable. Theoretically there could be infinitely many branded checkout domains, but a fairly fixed number of non-branded checkout domains.

Reply:
I wasn't sure whether to go with a whitelist or blacklist. Are there any other staging domains besides `give-stage2.cru.org`?

Reply:
The EP upgrade is currently on `give-stage2-next.cru.org` which will go away once we get closer to that project finishing.